packages/charting/src/components/Legends/shape.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { LegendShape } from './Legends.types';
+import { Points } from '../../utilities/utilities';
+
+export interface IShapeProps {
+  svgProps: React.SVGAttributes<SVGElement>;
+  pathProps: React.SVGAttributes<SVGPathElement>;
+  shape: LegendShape;
+  classNameForNonSvg?: string;
+}
+
+type PointPathType = {
+  [key: string]: string;
+};
+
+const pointPath: PointPathType = {
```

Comment:
In this case since you have a bunch of long path strings you can just disable the rule for the rest of the file like this:
```suggestion
/* eslint-disable @fluentui/max-len */
const pointPath: PointPathType = {
```